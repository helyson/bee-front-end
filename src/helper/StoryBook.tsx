import React from "react";
import "./../assets/styles/main.scss";

const StoryBookWrapper: React.FC = ({ children }) => {
  return <>{children}</>;
};

export { StoryBookWrapper };
