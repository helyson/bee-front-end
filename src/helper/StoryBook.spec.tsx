import React from "react";
import { cleanup, render } from "@testing-library/react";
import { StoryBookWrapper } from "./StoryBook";

describe(" <StoryBook />", () => {
  afterEach(cleanup);
  test("should be render icon", () => {
    const { container } = render(<StoryBookWrapper />);
    expect(container).toMatchSnapshot();
  });
});
