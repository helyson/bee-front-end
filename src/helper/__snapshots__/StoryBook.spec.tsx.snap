// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` <StoryBook /> should be render icon 1`] = `<div />`;
